.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+**Do you want to request a *feature* or report a *bug*?**  
+(If this is a *usage question*, please **do not post it here** - post it on [Stack Overflow](http://stackoverflow.com/questions/tagged/redux) instead. If this is not a "feature" or a "bug", or the phrase "How do I...?" applies, then it's probably a usage question.)
```

Comment:
Actually, I'd point out that this isn't even part of the docs - it's just an issue template.